metagraph/src/graph/representation/succinct/boss_chunk_construct.hpp

```diff
@@ -28,7 +29,8 @@ class IBOSSChunkConstructor : public IGraphChunkConstructor<BOSS::Chunk> {
                double memory_preallocated = 0,
                mtg::kmer::ContainerType container_type = mtg::kmer::ContainerType::VECTOR,
                const std::filesystem::path &swap_dir = "/tmp/",
-               size_t max_disk_space_bytes = 1e9);
+               size_t max_disk_space_bytes = 1e9,
+               const BuildCheckpoint& checkpoint = BuildCheckpoint(false, "", 2));
```

Comment:
```suggestion
               const BuildCheckpoint &checkpoint = BuildCheckpoint(false, "", 2));
```

Reply:
Maybe make the empty prefix mean it's disabled, so you don't have to pass true/false?